test/integration/operator_policy_complianceconfig_test.go

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	policiesv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test OperatorPolicy complianceConfig",
+	Ordered, Label("BVT"), func() {
```

Comment:
I'd rather this be an SVT test since it's a more advanced feature.
```suggestion
	Ordered, func() {
```